dias/analyzers/daily_ringmaps.py

```diff
@@ -0,0 +1,202 @@
+"""Analyzer to retrieve and store daily ringmaps."""
+import numpy as np
+import requests
+import msgpack
+from datetime import datetime
+from os import path
+import h5py
+
+from dias import CHIMEAnalyzer
+from dias.exception import DiasDataError
+from dias.utils import str2timedelta, datetime2str
+from caput import config
+from ch_util import ephemeris as ephem
+from bitshuffle import h5
+
+TIME_DTYPE = np.dtype([("fpga_count", "<u8"), ("ctime", "<f8")])
+POL_LABELS = ["XX", "XY", "YX", "YY"]
+
+
+class DailyRingmapAnalyzer(CHIMEAnalyzer):
+    """
+    Analyzer to retrieve and store daily ringmaps.
+
+    Metrics
+    -------
+    dias_task_daily_ringmaps_maps_total
```

Comment:
```suggestion
    dias_task_<task_name>_maps_total
```

The task name is a part of the metric name and it is configurable (I think it's the name of the task config file).